policyengine_uk/variables/gov/hmrc/tax_free_childcare/tax_free_childcare.py

```diff
@@ -3,43 +3,37 @@
 
 class tax_free_childcare(Variable):
     value_type = float
-    entity = BenUnit
+    entity = Person
     label = "Tax-Free Childcare government contribution"
     documentation = "The amount of government contribution provided through the Tax-Free Childcare scheme"
     definition_period = YEAR
     unit = GBP
     defined_for = "tax_free_childcare_eligible"
```

Comment:
This will work but just to raise here that this is implicitly casting the benefit unit level variable down to people. No action needed.